tracer/src/Datadog.Tracer.Native/environment_variables.h

```diff
@@ -135,6 +135,9 @@ namespace environment
 
     // Security Controls config.
     const shared::WSTRING security_controls_configuration = WStr("DD_IAST_SECURITY_CONTROLS_CONFIGURATION");
+
+    // Enables or disables the skip method body feature
+    const shared::WSTRING internal_skip_method_body_enabled = WStr("DD_INTERNAL_SKIP_METHOD_BODY_ENABLED");
```

Comment:
Is this really necessary 🤔 It looks like it was something you thought you would need when trying to figure out the issue, but it isn't actually necessary?

Reply:
What's more, it looks essentially dangerous, this doesn't seem like it's something that should be configurable 🤔